### DISCUSSION TITLE
What is the version of c++ in BitcrazeVM 2023?

### DISCUSSION THREAD

**Discussion by whY-note** [TinyMPC/TinyMPC#42] [Q&A]
What is the version of c++ in BitcrazeVM 2023?
I find the tinympc code which runs on BitcrazeVM 2023 will report an error

**Comment by xkhainguyen** [comment]
Hi, we did not use the Bitcraze VM for our experiments. Instead, we installed everything directly on our native Ubuntu environment, which is both convenient and fast. I would recommend doing the same.
